Makefile

```diff
@@ -8,11 +8,12 @@
 
 PACKAGE=acmart
 
-
-PDF = $(PACKAGE).pdf acmguide.pdf
+BIBLATEXFILES= $(wildcard *.bbx) $(wildcard *.cbx) $(wildcard *.dbx) $(wildcard *.lbx)
+SAMPLEBIBLATEXFILES=$(patsubst %,samples/%,$(BIBLATEXFILES))
 
 all:  ${PDF} ALLSAMPLES
 
+test: $(SAMPLEBIBLATEXFILES)
```

Comment:
This target, if intended to be included in this PR, should be marked as phony.

Reply:
May you try `make distclean` and the `make test` to see if the *.*bx files are properly copied into the samples directory?
Here is what should happen:

```
$ make test
cp acmauthoryear.bbx samples
cp acmnumeric.bbx samples
cp software.bbx samples
cp acmauthoryear.cbx samples
cp acmnumeric.cbx samples
cp acmauthoryear.dbx samples
cp acmnumeric.dbx samples
cp software.dbx samples
cp english-software.lbx samples
cp french-software.lbx samples
```

Reply:
sure, that works.

Is the intention to make `all` target depend on `test`?

Reply:
> Maybe the `make` we use have different ideas of the order of precedence of the rules?

It's possible, but my understanding is that when invoked without an explicit target, make picks up the first target only, that is, `all` in this case, `test` was never executed in my understanding.

But your recent fix indeed works.